.eslintrc.js

```diff
@@ -5,4 +5,7 @@
 
 module.exports = {
   extends: ['@loopback/eslint-config'],
+  rules: {
+    '@typescript-eslint/no-unused-vars': 'off',
```

Comment:
I am very reluctant to disable this rule for the entire example codebase. If it's too much work to fix your front-end code to not rely on unused variables, then please disable this rule selectively for the front-end files only. 

See how we are using `overrides` config section in our "master" eslint config here:

https://github.com/strongloop/loopback-next/blob/3e15bcada2aa4f4cf6c0440b54d0233880b32c03/packages/eslint-config/eslintrc.js#L158-L166
```
  overrides: [
    {
      files: ['**/*.js'],
      rules: {
        '@typescript-eslint/prefer-optional-chain': 'off',
        '@typescript-eslint/prefer-nullish-coalescing': 'off',
      },
    },
  ],
```